viewing_party/party.py

```diff
@@ -1,23 +1,105 @@
-# ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie = {"title": title, "genre": genre, "rating": rating}
+    else: 
+        movie = None
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data

```

Comment:
The same bug as `add_to_watched` applies here too. But as our tests didn't catch it, that's on us more than you.